pagopa/mui-italia
# feat(PN-9571): adaptation of drag and drop component to WCAG 2.2 standards

## Short description
Added CTA in _SingleFileInput_ component

### Preview
![image](https://github.com/pagopa/mui-italia/assets/73442810/016b97c3-e4ac-4485-9e0b-2b45e9aecfdc)

## List of changes proposed in this pull request
- Added button to component to make CTA more accessible
- Added prop for copy of button

## Product
Piattaforma Notifiche -> SEND

## How to test
Run storybook and go to SingleFileInput component to check that box has button to upload file
